cookbook/prompt_management_langchain.ipynb

```diff
@@ -161,28 +161,18 @@
       ]
```

Comment:
<p>This will break the notebook, the following needs to remain as a code block</p><p> </p>

<pre class="ql-syntax" spellcheck="false"># Get current production version of prompt 
langfuse_prompt = langfuse.get_prompt("event-planner")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/589/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>